src/libos/src/fs/async_file_handle.rs

```diff
@@ -68,6 +68,25 @@ impl AsyncFileHandle {
         Ok(total_len)
     }
 
+    pub async fn preadv(&self, bufs: &mut [&mut [u8]], offset: usize) -> Result<usize> {
```

Comment:
I think the `pread` and `pwrite` method should also be implemented in `AsyncFileHandle.`

Reply:
Makes sense，I will add a new commit for this PR